frontend/compositions/visualizations/map/BaseMap.tsx

```diff
@@ -120,19 +85,30 @@ export default function BaseMap(props: BaseMapProps) {
       .classed("state", true)
       .attr("title", (d) => d.properties.name)
       .attr("d", path)
-      .attr("filter", "url(#strokeShape)")
-      .attr("fill", (d: Feature) => {
-        const countIncidents = data.filter((i) => d.id === i.state).length
-        return colorGradient(valueScale(countIncidents))
-      })
-      // .attr("pointer-events", "all")
+      .attr("fill", (d: Feature) => "lightblue")
+      .attr("stroke", "white")
       .attr("cursor", "pointer")
+      .attr("pointer-events", "all")
 
     return () => {
-      defs.remove()
+      // defs.remove()
       paths.remove()
     }
-  }, [data, valueScale, path, colorGradient, geoData, svg])
-
-  return <svg id="map" viewBox={`0, 0, 1200, 700`} ref={baseMapRef}></svg>
+  }, [valueScale, path, colorGradient, geoData, svg])
+
+  return (
+    <svg id="map" viewBox={`0, 0, 1200, 700`} ref={baseMapRef}>
+      <rect
+        height={100}
+        width={100}
```

Comment:
do we want to do this through CSS instead?

Reply:
this is a chewing-gum fix for a temp bug having to do with loading the temp data.  should be gone when we hook up the real data sources

Reply:
sweet good to know